tasks/copy_to_glacier/README.md

```diff
@@ -127,16 +176,17 @@ The `copy_to_glacier` lambda function expects that the input payload has a `gran
   }
 }
 ```
-
+From the json file, the `filepath` shows the current S3 location of files that need to be copied over to glacier ORCA S3 bucket such as `"filename": "s3://orca-sandbox-protected/MOD09GQ/006/MOD09GQ.A2017025.h21v00.006.2017034065109.hdf"`.
 **Note:** We suggest that the `copy_to_glacier` task be placed any time after the `MoveGranulesStep`. It will propagate the input `granules` object as output, so it can be used as the last task in the workflow.
+See `schemas/input.json` for more information.
 
 
```

Comment:
I would add a config section and point to our config.json schema and talk about the configuration needed for the task through CMA.

Reply:
added